style.css

```diff
@@ -0,0 +1,253 @@
+/* CONTAINER - it's a grid*/
+
+#container {
+    position: relative;
+    background: linear-gradient();
+    padding: 0px;
+    margin: 0px;
+    height: fit-content;
+    display: grid;
+    grid-template-columns: 2fr 2fr 2fr 2fr;
+    grid-template-rows: .5fr 1fr 1fr 1fr;
+    grid-gap: 1px;
+    grid-template-areas: 
+    "header header header header"
+    "features features features features"
+    "main main main sidebar"
+    "main main main sidebar"
+    "footer footer footer footer";
+}
```

Comment:
This should be specific for each individual page rather then using the same one for every page.